scripts/ci/maintenance/semantic_release_compute_next.py

```
@@ -215,6 +233,82 @@ def read_pyproject_version() -> str:
+                    "application/vnd.docker.distribution.manifest.v2+json"
+                ),
+                "Authorization": f"Bearer {token}",
+            }
+            resp = client.head(
+                f"https://ghcr.io/v2/{image}/manifests/{tag}",
+                headers=headers,
+            )
+            if resp.status_code != 200:
+                return ""
+            return str(resp.headers.get("Docker-Content-Digest", ""))
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)